test/unit/gateways/migs_test.rb

```diff
@@ -3,18 +3,22 @@
 class MigsTest < Test::Unit::TestCase
   def setup
     @gateway = MigsGateway.new(
-                 :login => 'login',
-                 :password => 'password',
-                 :secure_hash => '76AF3392002D202A60D0AB5F9D81653C'
+                 login: 'login',
+                 password: 'password',
+                 secure_hash: '76AF3392002D202A60D0AB5F9D81653C',
+                 advanced_login: 'advlogin',
+                 advanced_password: 'advpass'
                )
-
     @credit_card = credit_card
     @amount = 100
+    @authorization = '2070000742'
+    @tx_source = 'MOTO'
 
     @options = {
-      :order_id => '1',
-      :billing_address => address,
-      :description => 'Store Purchase'
+      order_id: '1',
+      billing_address: address,
+      description: 'Store Purchase',
+      return_url: 'http://localhost:8080/payments/return'
```

Comment:
I think you added ```return_url``` previously because you were testing ```purchase_offsite_url```. Since we are no longer modifying that method and ```return_url``` isn't used anywhere else, we should remove it from the ```@options``` hash here.

Reply:
👍 Done!